CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Bump flatcar to `3510.2.0`.
+- Bump etcd to `3.5.8`.
+- Bump `cilium-app` to `0.9.2`.
```

Comment:
Btw. you haven't bumped cilium-app in this PR

Reply:
But I see, this was already merged previously. I guess it's just a note 👍🏻

Reply:
yes it was a failed merge, thanks for spotting it